tox.ini

```diff
@@ -54,10 +54,8 @@ description = Live development: build the Sphinx docs with autoreloading enabled
 deps = {[testenv:docs]deps}
 commands_pre =
     {[testenv:docs]commands_pre}
-# https://github.com/sphinx-doc/sphinx-autobuild/issues/170
-change_dir = {tox_root}/docs
 commands =
-    sphinx-autobuild ./ ./_build/html --watch . --port 8000
+    sphinx-autobuild docs/ docs/_build/html --watch ops/ --port 8000 {posargs}
```

Comment:
Does this mean that the live reload will now update if the content changes, rather than just the config?

Reply:
it should be on both, content and settings.
`--watch` means additional directories to watch, according to usage.